WalletWasabi.Fluent/ViewModels/Wallets/Buy/Workflows/ShopinBit/1 - Initial/CountryStep.cs

```diff
@@ -33,10 +33,10 @@ public CountryStep(Conversation conversation, IReadOnlyList<Country> countries,
 	protected override IEnumerable<string> BotMessages(Conversation conversation)
 	{
 		// Assistant greeting, min order limit
-		yield return $"Hello, I am your chosen {GetAssistantName(conversation)}. At present, we focus on requests where the value of the goods or services is at least $1,000 USD.\nPlease keep in mind that we can only organize legal products.";
```

Comment:
I would keep the greeting here. We were working hard on making the user feel that they actually chatting with a real person. So the greeting adds for the UX here.
Also, I would merge the two sentences to make it shorter like:

```
$"Hello, I am your chosen {GetAssistantName(conversation)}. Keep in mind that for now, we only accept requests for legal goods or services of at least $1,000 USD.
```

Reply:
That's fine for me, I just think it is repetitive to have the Assistant name 3 times in the same chat.

Reply:
Please make the change and I will merge the PR

Reply:
Done. [30bab56](https://github.com/zkSNACKs/WalletWasabi/pull/12105/commits/30bab5652041770a971fb738a659ff4140b613eb)